components/Tabbed/index.vue

```diff
@@ -12,16 +12,34 @@ export default {
   },
 
   data() {
-    return { tabs: null };
+    return {
+      tabs:            [],
+      initialTabOrder: []
+    };
+  },
+
+  computed: {
+    filteredTabs() {
```

Comment:
arrangedTabs or sortedTabs or something like that, they're not being filtered

Reply:
Updated